sdks/go/examples/wordcount/wordcount.go

```diff
@@ -110,16 +110,16 @@ func init() {
 }
 
 var (
-	wordRE            = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
-	empty             = beam.NewCounter("extract", "emptyLines")
-	small_word_length = flag.Int("small_word_length", 9, "small_word_length")
-	small_words       = beam.NewCounter("extract", "small_words")
-	lineLen           = beam.NewDistribution("extract", "lineLenDistro")
+	wordRE          = regexp.MustCompile(`[a-zA-Z]+('[a-z])?`)
+	empty           = beam.NewCounter("extract", "emptyLines")
+	smallWordLength = flag.Int("small_word_length", 9, "smallWordLength")
```

Comment:
Note that the 2nd string parameter is the flag's description when --help is called. I should have caught this in the initial change, but we should put a proper description of the flag's purpose there instead of repeating the variable.